package.json

```diff
@@ -55,7 +55,7 @@
     "merge-stream": "2.0.0",
     "pkg-dir": "4.2.0",
     "prismjs": ">=1.24.0",
-    "spec-up": "^0.10.6",
+    "spec-up": "github:brianorwhatever/spec-up#fix-refs",
```

Comment:
we have a newly forked version of spec-up under toip, would this be a good place to merge in your changes, and create the upstream PR from there?

That way we can incubate the feature in toip while waiting for it to go to DIF?